src/usb-device.adb

```diff
@@ -159,8 +159,7 @@ package body USB.Device is
             return Handled;
 
          when others =>
-            raise Program_Error with "Descriptor not implemented " &
-              Desc_Type'Img;
+            Put_Line ("Descriptor not implemented:" & Desc_Type'Img);
```

Comment:
We don't want any call to `Put_Line` if `Verbose` is not `True`.

Reply:
`Put_Line` in this file already checks if `Verbose` is set.

Reply:
@Fabien-Chouteau please suggest how to proceed.